README.md

```diff
@@ -117,7 +117,23 @@ Check out how in the INSTALL section.
 
 3) Edit config.cfg to suit your needs (see [configuration](#configuration)). `acme-dns` will read the configuration file from `/etc/acme-dns/config.cfg` or `./config.cfg`
 
-4) Run acme-dns. Please note that acme-dns needs to open a privileged port (53, domain), so it needs to be run with elevated privileges.
+4) If your system has systemd, you can optionally install acme-dns as a service so that it will start on boot and be tracked by systemd. This also allows us to add the `CAP_NET_BIND_SERVICE` capability so that acme-dns can be run by a user other than root.
+
+    1) Make sure that you have moved the configuration file to `/etc/acme-dns/config.cfg` so that acme-dns can access it globally.
+
+    2) Move the acme-dns executable from `~/go/bin/acme-dns` to `/usr/local/bin/acme-dns` (Any location will work, just be sure to change `acme-dns.service` to match)
```

Comment:
Oh, I'm sorry, I was probably unclear in my previous comment. I meant the *source* directory, as people who didn't build it themselves, but instead downloaded a binary release from https://github.com/joohoi/acme-dns/releases will not have the executable in `~/go/bin/acme-dns`.

Reply:
Oh sorry I completely misunderstood what you meant! I was just trying to match the first few steps of that installation section since step 2 mentions building the executable. I can change it though!

Reply:
No, you are absolutely correct. We should make a different section for installing using the binary release (and possibly doing this after we have the installer script in place)